addons/sys_radio/fnc_isBaseClassRadio.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: ACRE2Team
+ * Returns whether a classname is a base radio classname. This function exists because it is used frequently and config lookup is expensive.
+ *
+ * Arguments:
+ * 0: Classname <STRING>
+ *
+ * Return Value:
+ * Is Unique <BOOL>
+ *
+ * Example:
+ * ["acre_prc152_id_1"] call acre_sys_radio_fnc_isBaseClassRadio
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _isBaseClassRadio = false;
+
+if (HASH_HASKEY(GVAR(radioBaseClassCache),_this)) then {
```

Comment:
`if-exitwith` is faster than `if-then-else` in this case, as it will only cache it on first call and then just needs the value.